examples/landspill/download_topo.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin python3
```

Comment:
Recommend using `#!/usr/bin/env python` instead to include greater compatibility.

Reply:
What I am worried about is the use of `urllib.request` module with Python 2. I haven't fully tested the script with Python 2 environments. And I know some Linux systems use Python 2 as the default.

I'll test it with Python 2 and see if I can make it work if there's something incompatible with Python 2.

Reply:
BTW, yes, I should use `/usr/bin/env` instead of `/usr/bin`. Using `/use/bin` is a bug.

Reply:
Done in 61455d9.